src/box/lua/net_box.c

```diff
@@ -815,8 +823,9 @@ netbox_communicate(int fd, struct ibuf *send_buf, struct ibuf *recv_buf,
  * Returns 0 and a decoded response header on success.
  * On error returns -1.
  */
-int
+static int
 netbox_send_and_recv_iproto(int fd, struct ibuf *send_buf,
+			    struct fiber_cond *on_send_buf_empty,
```

Comment:
Please be consistent about argument order. If on_send_buf_empty follows send_buf it should follow in all functions, including netbox_communicate and netbox_iproto_loop.

Reply:
fixed